src/osdc/Objecter.cc

```diff
@@ -1214,9 +1215,10 @@ void Objecter::handle_osd_map(MOSDMap *m)
 	update_pool_full_map(pool_full_map);
 
 	// check all outstanding requests on every epoch
```

Comment:
might want to update the comment accordingly, like

> collect removed snap in every past osdmaps

Reply:
updated it